handlers/namespaces.go

```diff
@@ -0,0 +1,26 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/swift-sunshine/swscore/kubernetes"
+	"github.com/swift-sunshine/swscore/log"
+)
+
+func NamespaceList(w http.ResponseWriter, r *http.Request) {
+	istioClient, err := kubernetes.NewClient()
+	if err != nil {
+		log.Error(err)
+		RespondWithError(w, 500, err.Error())
```

Comment:
Nitpicking and no blocker for this PR.
Probably HTTP codes are defined in some constant, I remember they are in some place under the net/http.

Reply:
indeed: `http.StatusInternalServerError`.
By the way, we could use the standard lib here: 
`http.Error(w, err.Error(), http.StatusInternalServerError)`

Reply:
About this...if the error is provided by client I prefer to do
```go
RespondWithJSON(w, int(err.(errors.APIStatus).Status().Code), err)
```

If you see the error provided by the client you can see the full error with message and the status code. 

When the error is controlled by us,Why not use the model error of client (	"k8s.io/apimachinery/pkg/api/errors")

```go 
return nil, errors.NewNotFound(schema.GroupResource{Resource: "namespaces"}, namespace)
```

Reply:
Forwarding the k8s client error code might not always be a good idea. For instance, a "bad request" returned by k8s server does not mean "bad request" from our UI client point of view.

Reply:
@aljesusg the err returned but the kubernetes layers hould provide that.
From handler perspective there should be an abstraction to the inner layer.
And I guess the same description would be provided in the message.

Reply:
We can control the errors kind bad_request @jotak, my point is use the same models and not do the job again.

Reply:
@aljesusg I guess we can find something in the middle, to not break the agreement to have a intermediate model we may want to control, and this could be just to cast the err as well.
So, at same way we cast the kubernetes model, it can be done the same to mark some specific use cases we might want to handle.
If not, I understand that the discussion is if we forward the full model from the k8s or to create a intermedate one.
So, I prefer to maintain our model, and perhaps add @aljesusg suggestion, in the cast/models layer.

Reply:
It was an idea but +1 @lucasponce